packages/@tinacms/toolkit/src/packages/fields/plugins/BlocksFieldPlugin.tsx

```diff
@@ -164,7 +166,16 @@ const Blocks = ({ tinaForm, form, field, input }: BlockFieldProps) => {
 
                   const itemProps = (item: object) => {
                     if (!template.itemProps) return {}
-                    return template.itemProps(item)
+                    if (typeof template.itemProps === 'function') {
+                      return template.itemProps(item)
+                    } else {
+                      const plugins = cms.plugins
+                        .all('itemProps')
+                        .filter(
```

Comment:
`PluginTypeManager` has a more succinct way to get a specific named plugin:

`cms.plugins.getType('itemProps').find(template.itemProps.name)`